brouter-core/src/main/java/btools/router/RoutingParamCollector.java

```diff
@@ -206,6 +206,11 @@ public void setParams(RoutingContext rctx, List<OsmNodeNamed> wplist, Map<String
           rctx.roundtripDistance = Integer.valueOf(value);
         } else if (key.equals("roundtripDirectionAdd")) {
           rctx.roundtripDirectionAdd = Integer.valueOf(value);
+        } else if (key.equals("roundTripPoints")) {
```

Comment:
@afischerdev The other round trip parameters have "roundtrip" in lower case:
`roundtripDistance` and `roundtripDirectionAdd`.

The `roundTripPoints` has the "trip" starting in upper case. We could make all of them similar?